resolwe/flow/migrations/0016_refactor_relations_2.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.14 on 2018-08-06 02:16
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def position_to_label(apps, schema_editor):
+    RelationPartition = apps.get_model('flow', 'RelationPartition')
+    for partition in RelationPartition.objects.all():
+        if partition.position:
+            partition.label = partition.position
```

Comment:
Do you expect issues with the type of old `partition.position`? This was JSONField but the new `partition.label` accepts strings. I suggest you check existing values in DB.